azkaban-common/src/test/java/azkaban/executor/ContainerizedDispatchImplTest.java

```diff
@@ -98,152 +100,139 @@ public void setup() throws Exception {
 
   @Test
   public void testFetchAllActiveFlows() throws Exception {
-    ContainerizedDispatchImpl containerizedDispatchImpl = initializeContainerizedDispatchImpl();
+    initializeContainerizedDispatchImpl();
```

Comment:
This should be part of the setUp function.

Reply:
Moving it to setup, fails the test cases due to its multithreaded nature. That is why initialization is kept in each test method as its scope will be limited to method. If we move it to setup, then it will be freed gc is triggered.